### PR TITLE
WEB: Formalize workgroups

### PR DESCRIPTION
**Pull request by datapythonista**
Adding to our `Team` page the different workgroups we more or less already have, with their responsibilities. So, things become more formal, and it's easier to manage expectations.

I wonder if it'd be useful to add the email addresses to contact the different workgroups here. Open to opinions.

CC: @pandas-dev/pandas-core

### CODE REVIEW

**Review by MarcoGorelli — APPROVED** [review]
No objections

**Review by mroeschke — APPROVED** [review]

**Review by attack68 — APPROVED** [review]
good idea

**Review by lithomas1 — APPROVED** [review]
LGTM. It might be nice to draft up a charter as a follow up to clarify each groups resposibilities.

File: web/pandas/about/team.md
Comment (WillAyd) [inline web/pandas/about/team.md:54]: Might be misreading but shouldn't this be `workgroup.coc`?

**Comment by mroeschke** [comment]
> Is it necessary to backport? The website is deployed from main anyway?

Typically we have been backporting the `WEB` changes, but up to @datapythonista

**Comment by datapythonista** [comment]
For what I know, it doesn't add any value to backport website changes. No harm either, but as Patrick says, the website is directly published from main, and independent of releases. Unless there is anything I'm missing, I wouldn't backport web PRs.

**Comment by Dr-Irv** [comment]
> I wonder if it'd be useful to add the email addresses to contact the different workgroups here. Open to opinions.

I would suggest that you do not publish email addresses.  It might be worthwhile to create email addresses for each workgroup (if that is possible).

**Comment by datapythonista** [comment]
> I would suggest that you do not publish email addresses. It might be worthwhile to create email addresses for each workgroup (if that is possible).

We already have the email addresses and we're using them.

**Comment by Dr-Irv** [comment]
> 
> We already have the email addresses and we're using them.

So I think you should include the email addresses of the workgroups in this doc.

**Comment by datapythonista** [comment]
Sounds good. I think the coc one has been in the website for some years, and spam hasn't been a problem, so I think it can be useful to have the email addresses in the website, for both the rest of core devs and the community.

I just invited the members of the finance workgroup to their list, which was created but members hasn't been added until now.

This is how the final version looks like:

workgroups

**Comment by noatamir** [comment]
May I suggest sorting the names on this page alphabetically, by the first name? 
- Why sort? When things are listed, there's a tendency to associate importance unless a neutral ordering is provided.
- It will provide a better onboarding experience for new team members, who will tend to be added at the bottom otherwise. 
- Why first name? It's more inclusive to folks that have more than one last name and aren't accustomed to last name sorting. 
- Leads are already marked in brackets, so their order isn't essential?!

**Comment by MarcoGorelli** [comment]
> For what I know, it doesn't add any value to backport website changes. No harm either, but as Patrick says, the website is directly published from main, and independent of releases. Unless there is anything I'm missing, I wouldn't backport web PRs.

Isn't that only the case for the `dev` version of the docs? If we don't backport other things, then the `stable` version won't show the changes until 2.0.0

**Comment by phofl** [comment]
Ah no, there is a difference between website and docs.

The website (everything in web) is deployed from main (I think once a day but not sure). The docs are independent of that

**Comment by datapythonista** [comment]
> May I suggest sorting the names on this page alphabetically, by the first name?

The sorting is by seniority in the workgroup. I think this is more useful than by first name. Also, the lead is automatically assigned to the first in the list, so it makes things more complex is we change the sorting.

**Pull request merged**
